### ISSUE TITLE
[Pro] Allow mounting dashboard routes without loading the whole schema

### ISSUE THREAD

**Issue by such** [rmosolgo/graphql-ruby#3868]
**Is your feature request related to a problem? Please describe.**

In order to mount dashboard in the application it is advised to:
```
Rails.application.routes.draw do
  # ...
  # Add the GraphQL::Pro Dashboard
  # TODO: authorize, see below
  mount MySchema.dashboard, at: "/graphql/dashboard"
end
```

While lazy loading the code (in development for instance), since `MySchema` defines the root type (query, mutation and subcription), it triggers the load of all Graphql type classes which may also triggers the load of some Active Record models in my case. This can makes the startup process of a Rails app slower than it should.

**Describe the solution you'd like**

We should be able to mount the routes without loading the whole GraphqlSchema.

**Describe alternatives you've considered**

I've tried [routes_lazy_routes](https://github.com/amatsuda/routes_lazy_routes) but it's broken by `devise-jwt`.

**Comment by rmosolgo** [comment]
Yeah, that makes sense. I did it this way at first because the dashboard needs a reference to the schema object somehow. But I could add an API that accepts a `class_name:` instead and and `const_get`s that `class_name` when it actually needs it. That would put off loading the schema until later! 

Let me know if that doesn't seem like it would work for you, otherwise, I'll take a look and follow up here when it's ready.

**Comment by such** [comment]
That would be perfect! Thank you

**Comment by rmosolgo** [comment]
:+1: I just released `graphql-pro` v1.21.2 which includes `GraphQL::Pro::Routes::Lazy.new(class_name)` for this purpose: 

https://graphql-ruby.org/pro/dashboard.html#lazy-loading-the-schema

(Besides `.dashboard`, it implements the other route methods, too, like [`.ably_webhooks_client`](https://graphql-ruby.org/subscriptions/ably_implementation#server).) 

Please give it a try and let me know if you run into any issues with it!

**Comment by such** [comment]
Amazing! Thank you!
